src/pages/Lecture/Lecture.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import LectureHeader from './LectureHeader';
+import styled from 'styled-components';
+import Video from './Video';
+import LectureFooter from './LectureFooter';
+import LectureSideBar from './LectureSideBar';
+
+function Lecture() {
+  return (
+    <LectureDiv>
+      <LectureMain>
+        <LectureHeader title="JSX 기본 문법 알아보기 (ii)" />
+        <Video />
+        <LectureFooter />
+      </LectureMain>
+      <LectureSideBar />
+    </LectureDiv>
+  );
+}
+
+const LectureDiv = styled.div`
```

Comment:
```suggestion
const Container = styled.div`
```
태그 이름 안쓰셔도 되고 바깥쪽 감싸는 컴포넌트는 그냥 Container라고 짓는게 편한거 같아요 어차피 이름 안겹쳐서 괜찮습니다